adhocracy-plus/assets/js/unload_warning.js

```diff
@@ -26,6 +26,7 @@ document.addEventListener('DOMContentLoaded', () => {
   }
 
   // This clashes with the embed plugin and so is commented out for now until we have time to fix
+  // It's not commented out??
```

Comment:
oooo so @goapunk un-commented it and I merged it but not sure how neither of us caught that, I think it was working after the vanilla refactor, does that sound right @goapunk?

Reply:
yes, I think we left it like this because it was working? or because we knew the embed was about to be removed ? :shrug: